app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,13 +12,14 @@ class VerifyCsrfToken extends Middleware
      * @var array
      */
     protected $except = [
-        'editmp',
-        'addmp',
-        'editmember',
-        'editproject',
-        'editproject/{id}',
-        'create' ,
-        'createproject'
-        
+        'project/update',
```

Comment:
Disabling Cross Site Request Forgery for these routes does not seem wise. Basically they are no longer secure now.
https://laravel.com/docs/5.5/csrf